mimic/core.py

```diff
@@ -55,9 +58,12 @@ def __init__(self, clock, apis, domains=()):
     @classmethod
     def fromPlugins(cls, clock):
         """
-        Create a :obj:`MimicCore` from all :obj:`IAPIMock` plugins.
+        Create a :obj:`MimicCore` from all :obj:`IAPIMock` and
+        :obj:`IAPIDomainMock` plugins.
         """
-        return cls(clock, list(getPlugins(IAPIMock, plugins)))
+        all_plugins = getPlugins(IAPIMock, plugins)
```

Comment:
Minor nitpick: `all_` is a bad prefix, it should be like, `service_catalog_` or something, since it clearly isn't *all* plugins.  That won't stop me from merging, though.